test/test_torch.py

```diff
@@ -12443,6 +12443,29 @@ def test_sum_out(self, device, dtype):
         torch.sum(x, (2, 1), out=res2)
         self.assertEqual(res1, res2)
 
+    def test_memory_format_type_shortcuts(self, device):
+        def input_generator_fn(device):
+            return torch.randn((10, 3, 32, 32), device=device, dtype=torch.float32).clamp(0, 1).round().contiguous(memory_format=torch.channels_last)
```

Comment:
Does the tensor really need to be this big in the test?

Reply:
Not really, will make it smaller.

Reply:
Fixed in #27891